hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMBucketCreateRequestWithFSO.java

```diff
@@ -182,4 +183,36 @@ protected void doValidateAndUpdateCache(String volumeName, String bucketName,
     verifySuccessCreateBucketResponse(omClientResponse.getOMResponse());
 
   }
+
+  @Test
+  public void testNonS3BucketNameAllowedForFSOWhenStrictDisabled() throws Exception {
+    // Arrange
+    ozoneManager.getConfiguration().setBoolean(
+        OMConfigKeys.OZONE_OM_NAMESPACE_STRICT_S3, false); // 如果常數名稱不同，改成實際的 key
```

Comment:
Nit: Please remove the mandarin comment (since some community members may not understand it) and the "Arrange" comment also can be removed.